File: app/src/main/java/com/xyoye/dandanplay/utils/net/service/TorrentRetrofitService.java
package com.xyoye.dandanplay.utils.net.service;

import io.reactivex.Observable;
import okhttp3.RequestBody;
import okhttp3.ResponseBody;
import retrofit2.http.Body;
import retrofit2.http.POST;

/**
 * Created by xyoye on 2020/2/23.
 */

public interface TorrentRetrofitService {

    @POST("/Magnet/Parse")
    Observable<ResponseBody> downloadTorrent(@Body RequestBody requestBody);
}

File: common_component/src/main/java/com/xyoye/common_component/network/service/TorrentRetrofitService.kt
package com.xyoye.common_component.network.service

import okhttp3.RequestBody
import okhttp3.ResponseBody
import retrofit2.http.Body
import retrofit2.http.POST

/**
 * Created by xyoye on 2021/1/3.
 */

interface TorrentRetrofitService {

    @POST("/Magnet/Parse")
    suspend fun downloadTorrent(@Body requestBody: RequestBody): ResponseBody
}